generic3g/FieldDictionaryItem.F90

```diff
@@ -107,4 +107,12 @@ pure function get_aliases(this) result(aliases)
       aliases = this%aliases
    end function get_aliases
 
+   pure function get_regrid_method(this) result(regrid_method)
+      class(FieldDictionaryItem), intent(in) :: this
+      type(ESMF_RegridMethod_Flag), allocatable :: regrid_method ! result
+      if (allocated(this%regrid_method)) then
+         allocate(regrid_method, source=this%regrid_method)
+      end if
```

Comment:
Needs a default return value
```suggestion
      regrid_method = ' '
      if (allocated(this%regrid_method)) then
           regrid_method = this%regrid_method
      end if
```

Reply:
I made a mess there. Actually, in several places. I'll fix it.

Reply:
I think I fixed this particular issue.